### DISCUSSION TITLE
Significant slowness in sending files

### DISCUSSION THREAD

**Discussion by david-lev** [localsend/localsend#363] [Discussion (English, International)] [answered]
I'm not sure that this is indeed a problem, so I didn't open an issueץ
When I transfer files from my Thinkpad (Pop!_OS) to the Samsung Galaxy S21 (Android 13) (the connecting network is another Galaxy S21 device that shares a hotspot for the Thinkpad and the S21), the transfer speed is very slow: 4.5 MB/s.
Is this a reasonable speed or is there a way to speed it up?
![image](https://user-images.githubusercontent.com/42866208/230050774-aa0ed906-80ab-4f3f-a24f-e90c7174ea5f.png)

**Comment by Tienisto** [comment, marked as answer]
You can get around 30-50 MB/s in 5Ghz Wifi. On 2,4 Ghz the speed should be around 5-10 MB/s depending on the interference of other 2,4 Ghz users.

Edit: Because you are using hotspot on S21, it should be very fast. I don't know why it isn't in your case. Maybe your Thinkpad is falling back to 2,4 Ghz? Another possible causes are (1) S21 has slow write-to-disk speed, (2) frequency is heavily used by other members (3) long distance, but this shouldn't be the case

**Reply by david-lev** [reply]
It turns out that the shared hotspot was on 2.4hz
I changed to 5hz, moved the band to 5hz in the Thinkpad network settings and it worked.. Thanks

**Comment by david-lev** [comment]
Wow.. 4x speed.. Amazing
I think it should be highlighted in the README or some other prominent location
Thanks for the help and this excellent tool!

![image](https://user-images.githubusercontent.com/42866208/230056915-78af14ec-bc2d-4a5b-93ca-76c9d9343c0d.png)

**Comment by DelidExpert** [comment]
![localsend_app_eoYGBwF8In](https://github.com/user-attachments/assets/eaffe427-5b40-4371-a75e-da096fee1fc6)
Sheeeeeeeeeeeeeesh, thanks for the tip!

**Reply by SeryiBaran** [reply]
> ![localsend_app_eoYGBwF8In](https://private-user-images.githubusercontent.com/149346328/370954106-eaffe427-5b40-4371-a75e-da096fee1fc6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjkwMDAwNTUsIm5iZiI6MTcyODk5OTc1NSwicGF0aCI6Ii8xNDkzNDYzMjgvMzcwOTU0MTA2LWVhZmZlNDI3LTViNDAtNDM3MS1hNzVlLWRhMDk2ZmVlMWZjNi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAxNVQxMzQyMzVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04YmI1MTg4ZjUyMzQ2N2VjMWE5YmQ4MGI0ZmE5ZTMyY2IxZmY1YmU0OWE5ZjEwZjMxNTcyNjk4ODY3ZTBiZDk0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.cieCKa8X4PD1Q27SYKhOzuB7CN0PbTPBzB0nrAmVxQM) Sheeeeeeeeeeeeeesh, thanks for the tip!

HOW do you get 81 fckin MB/s? I cant get it on gigabit network - only 10MB/s (not because 100mbit net, its gbit, i checked!)!

**Reply by Tienisto** [reply]
Try disable encryption on all your devices in the advanced settings

**Reply by DelidExpert** [reply]
> > ![localsend_app_eoYGBwF8In](https://private-user-images.githubusercontent.com/149346328/370954106-eaffe427-5b40-4371-a75e-da096fee1fc6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjkwMDAwNTUsIm5iZiI6MTcyODk5OTc1NSwicGF0aCI6Ii8xNDkzNDYzMjgvMzcwOTU0MTA2LWVhZmZlNDI3LTViNDAtNDM3MS1hNzVlLWRhMDk2ZmVlMWZjNi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAxNVQxMzQyMzVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04YmI1MTg4ZjUyMzQ2N2VjMWE5YmQ4MGI0ZmE5ZTMyY2IxZmY1YmU0OWE5ZjEwZjMxNTcyNjk4ODY3ZTBiZDk0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.cieCKa8X4PD1Q27SYKhOzuB7CN0PbTPBzB0nrAmVxQM) Sheeeeeeeeeeeeeesh, thanks for the tip!
> 
> HOW do you get 81 fckin MB/s? I cant get it on gigabit network - only 10MB/s (not because 100mbit net, its gbit, i checked!)!

I suppose Wifi6, Snapdragon 8 gen 3 and UFS4.0 storage are all contributing factors as well.

![image](https://github.com/user-attachments/assets/1a31b760-785e-4e66-8ab1-f8fa24a6787a)

**Reply by SeryiBaran** [reply]
> Try disable encryption on all your devices in the advanced settings

Its give me ~70MB/s! Thanks!!! :heart:
